app/src/main/java/org/lineageos/twelve/viewmodels/TwelveViewModel.kt

```diff
@@ -123,6 +123,7 @@ abstract class TwelveViewModel(application: Application) : AndroidViewModel(appl
     }
 
     private fun Audio.toMediaItem() = MediaItem.Builder()
```

Comment:
Note to self: We have to ensure buildMediaItem matches this
Things go weirdly bad in case they don't match